Dockerfile.ocp

```diff
@@ -4,8 +4,14 @@ COPY . .
 RUN make build
 RUN make tools
 
+COPY /go/src/github.com/metal3-io/baremetal-operator/ocp/ocp_kustomization.yaml /go/src/github.com/metal3-io/baremetal-operator/config/crd/kustomization.yaml
```

Comment:
It should be  COPY /go/src/github.com/metal3-io/baremetal-operator/config/crd/ocp/ocp_kustomization.yaml

Reply:
That commit included a move of the ocp_kustomization.yaml to the path referenced here. Updated the commit to not do the move anymore.